src/explorer/weights/WeightSlider.js

```diff
@@ -2,30 +2,61 @@
 
 import React from "react";
 
-export const WEIGHT_SLIDER_MIN = -5;
-export const WEIGHT_SLIDER_MAX = 5;
+/**
+ * The Weight is the number that the WeightSlider actually provides to the
+ * instantiating component. The Weight varies in the range
+ * [0, 2**MAX_SLIDER_VALUE].
+ */
+export type Weight = number;
+
+/**
+ * The SliderPosition is the current position of the [range input] that the
+ * WeightSlider instantiates. It varies in the range [MIN_SLIDER, MAX_SLIDER].
```

Comment:
nit: maybe backtick `` `[MIN_SLIDER, MAX_SLIDER]` `` to contrast with
the Markdown link syntax `[range input]` used immediately both above and
below.

Reply:
Moved it elsewhere in the docs, also backticked it.